database/clickhouse/clickhouse_test.go

```diff
@@ -29,7 +30,7 @@ var (
 )
 
 func clickhouseConnectionString(host, port string) string {
-	return fmt.Sprintf("clickhouse://%v:%v?username=user&password=password&database=db&x-multi-statement=true&debug=false", host, port)
+	return fmt.Sprintf("clickhouse://%v:%v?username=user&password=password&database=db&x-multi-statement=true&x-table-engine=MergeTree&debug=false", host, port)
```

Comment:
Add a separate test for MergeTree. We should still run most tests with the default table engine.

Reply:
Refactored the test cases to run both with MergeTree and TinyLog please take a look, thanks!